Configuration/Commands.php

```diff
@@ -21,6 +21,9 @@
  * https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.3/Deprecation-89139-ConsoleCommandsConfigurationFormatCommandsPhp.html
  */
 return [
+    'kitodo:delete' => [
```

Comment:
I believe this addition has no affect any more. (See comment from Alexander or here https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/10.3/Deprecation-89139-ConsoleCommandsConfigurationFormatCommandsPhp.html)

In fact we can remove all entries and are still able to call the commands.

Reply:
I tried registering the command in `Configuration/Services.yaml` (https://docs.typo3.org/m/typo3/reference-coreapi/10.4/en-us/Configuration/Yaml/ServicesYaml.html) but it did not worked:

````
                                           
  Command "kitodo:delete" is not defined.  
                                           
  Did you mean one of these?               
      kitodo:harvest                       
      kitodo:index                         
      kitodo:reindex                                                                 

````